### DISCUSSION TITLE
Conditional if statement prompting frontmatter values

### DISCUSSION THREAD

**Discussion by andrewbartnik** [SilentVoid13/Templater#967] [Help]
I have a template with frontmatter that looks like this
---
type: <% await tp.system.suggester(["School", "Work", "Other"], ["School", "Work", "Other"])%>
work_type: await tp.system.suggester(["website", "github"], ["website", "github"])%>
school_type: await tp.system.suggester(["assignment", "project"])
---

I would like, upon selecting "Work" as type:, to be prompted to select from the work_type: options in the second line, while upon selecting ""school" to be prompted to the options in the 3rd line. I have tried using some variations on if statements, but have had no luck. I'm sure I'm missing something quite simple. Any suggestions?

**Comment by AB1908** [comment]
Can you look at some other examples in the discussions that use `<%*` commands? You essentially need to write JavaScript.

**Reply by andrewbartnik** [reply]
My apologies, very new to JavaScript. Did some digging and still can't figure it out. I'm sure I'm close here.

<img width="784" alt="Screen Shot 2022-12-27 at 1 36 19 PM" src="https://user-images.githubusercontent.com/63502749/209714513-3133c2dc-ece1-434f-a3d2-f0eaf2bbc97a.png">
